src/com/linkedin/parseq/Task.java

```diff
@@ -1300,9 +1300,9 @@
    *          user.getFirstName() + user.getLastName() + " working for " + company.getName());
    * </pre></blockquote>
    *
-   * If any of tasks passed in as a parameters fails then returned task will also fail immediately.
+   * If any of tasks passed in as a parameter fails then returned task will also fail immediately.
    * In this case returned task will be resolved with error from the first of failing tasks and other
-   * tasks will be cancelled (if possible).
+   * tasks will be cancelled if task has not been completed yet.
```

Comment:
replace with "they have not been completed yet"?

Also add:
"Tasks will be started in an arbitrary order."

Reply:
Done